daemon/cmd/daemon_main.go

```diff
@@ -466,9 +466,12 @@ func initializeFlags() {
 		option.FixedIdentityMapping, "Key-value for the fixed identity mapping which allows to use reserved label for fixed identities, e.g. 128=kv-store,129=kube-dns")
 	option.BindEnv(option.FixedIdentityMapping)
 
-	flags.Duration(option.IdentityChangeGracePeriod, defaults.IdentityChangeGracePeriod, "Time to wait before using new identity on endpoint identity change")
+	flags.Duration(option.IdentityChangeGracePeriod, defaults.IdentityChangeGracePeriod, "Time to wait before using new identity on endpoint identity change when k8s is not enabled")
 	option.BindEnv(option.IdentityChangeGracePeriod)
 
+	flags.Duration(option.IdentityRestoreGracePeriod, defaults.IdentityRestoreGracePeriod, "Time to wait before releasing unused restored CIDR identities during agent restart")
```

Comment:
Should we have `when k8s is not enabled` here as well?

Reply:
I put that text in the wrong place, sorry. And now that is removed again as we likely need a grace period also after k8s (not 100% sure so be safe for now).